src/patternfly/components/Button/button-bars-icon.hbs

```diff
@@ -0,0 +1 @@
+<i class="fas fa-bars"></i>
```

Comment:
for all the icons can you add `aria-hidden="true"`

Reply:
yes!